Jamm fixed to 0.3.2 (java 11 compatible)

fabianpiau opened the pull request.

whitingjr commented:
@fabianpiau any chance you can rebase your branch commits. To ensure your branch is consistent with the mainline branch ?

fabianpiau commented:
@whitingjr Sorry, a bit late but I have updated my PR. I saw there is a newer version of Jamm so used that one too. If there is still some issue with Java 11 and this PR fixed it, feel free to merge, otherwise just close it. I did not re-test it locally as in the meantime, I changed of project and don't use Cassandra anymore.

whitingjr commented:
@fabianpiau sorry if I misled you. I'm not a committer to this project.
